pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -2658,9 +2666,9 @@ ASTOpenStatement OpenStatement()  :
 ASTFetchStatement FetchStatement()  :
 {}
 {
-	 <FETCH> QualifiedName() [<BULK> <COLLECT>]
+	 <FETCH> QualifiedName() [<BULK> <COLLECT> {jjtThis.setUserData("BULK COLLECT"); }]
```

Comment:
We should add a boolean flag on the fetch statement: `jjtThis.setBulkCollect()` / `ASTFetchStatement.isBulkCollect()`.